HOWTO/xero.md

```diff
@@ -0,0 +1,71 @@
+#XERO
+
+##How to setup invoice template
+
+1. From the top menu navigate to **Settings > General Settings**
+2. From section **Features** navigate to **Invoice Settings**
+3. Pick an invoice to copy, from **Options** select **Copy**, set a name to identify invoice for your event, eg [Your event name, year]
+4. Customise your invoice:
+* Select **Options > Change Logo**
+Upload your image (maximum of 400 pixels wide and 120 pixels high).
+* Select **Options > Edit**
+Review selected fields (most of the fields/options should be preselected from the copied invoice). 
+
+Update the following fields:
+
+* **"Terms & Payment Advice (Invoice)"**
+Include your payment details, eg bank account, payment terms, etc. Sample:
+
+```
+Electronic payment preferred:
+Please, include invoice number in all electronic transfers.
+Please, email electronic transaction confirmation details to [your@mail.account]
+
+Direct deposit:
+  Bank:             [Your bank name] 
+  Account Name:     [Your bank account name]
+  BSB:              [BSB number]
+  Account Number:   [Your account number]
+
+For international payments:
+  SWIFT Code:       [Your bank SWIFT code]
+  Bank Address:     [Your bank address]
+
+If paying by cheque, please complete this payment advice, detach and post to the address provided.
```

Comment:
I suggest avoiding this. If a client can't make a direct deposit they'll let the conference know and at that point the council/subcommittee can figure out alternatives. In general though we want to discourage anything but direct deposits due to fees and complications.